lib/open-api-mocker-cli.js

```diff
@@ -39,13 +39,27 @@ const { argv } = require('yargs')
 		alias: 'v',
 		default: false
 	})
+	.option('quiet', {
+		type: 'boolean',
+		alias: 'q',
+		description: 'Suppress request output, only show errors',
+		default: false
+	})
 	.demandOption('schema')
 	.help();
 
 const logger = require('lllog')();
 
+if(argv.verbose && argv.quiet) {
```

Comment:
This can be achieved using `yargs` directly. Please see `option()` documentation [here](https://github.com/yargs/yargs/blob/HEAD/docs/api.md#optionkey-opt), and check the `conflicts` option.

It should be set on `verbose` and `quiet` options.